src/app/src/main/java/com/cmput301f21t09/budgetprojectname/HabitEventModel.java

```diff
@@ -12,22 +12,22 @@
 public class HabitEventModel{
     private final String ID; // TODO: change type depending on Firebase ID type
     private final String location; // TODO: change to Location once this is implemented
-    private final String description;
+    private final String comment;
     private final Date date;
     private final Image image;
 
-    HabitEventModel(String location, Date date, String description, Image image) {
+    HabitEventModel(String location, Date date, String comment, Image image) {
```

Comment:
Please add Javadoc, but it looks excellent otherwise :)❤️

Reply:
Added it in another PR!